Modules/itertoolsmodule.c

```diff
@@ -174,7 +174,7 @@ static PyMethodDef groupby_methods[] = {
 };
 
 PyDoc_STRVAR(groupby_doc,
-"groupby(iterable[, keyfunc]) -> create an iterator which returns\n\
+"groupby(iterable, key=None[, keyfunc]) -> create an iterator which returns\n\
```

Comment:
I think you need to remove the `keyfunc` parameter here. `groupby` supports just `iterable` and `key` as arguments